src/region.js

```diff
@@ -156,31 +156,19 @@ const Region = MarionetteObject.extend({
     // always restore the el to ensure the regions el is present before replacing
     this._restoreEl();
 
-    const parent = this.el.parentNode;
-
-    parent.replaceChild(view.el, this.el);
+    this.$el.replaceWith(view.el);
     this._isReplaced = true;
   },
 
   // Restore the region's element in the DOM.
   _restoreEl() {
-    if (!this.currentView) {
-      return;
-    }
-
     const view = this.currentView;
-    const parent = view.el.parentNode;
 
-    if (!parent) {
-      return;
+    if (view && view.el && view.el.parentNode) {
+      // Replace using library to ensure proper cleanup.
+      view.$el.replaceWith(this.el);
+      this._isReplaced = false;
     }
-
-    parent.replaceChild(this.el, view.el);
-    this._isReplaced = false;
-  },
-
-  isReplaced() {
```

Comment:
I didn't see it get used anywhere in the code base. I could have it get used where I saw it should have been used. I'll do that